packages/manager/apps/container/src/container/nav-reshuffle/sidebar/style.module.scss

```diff
@@ -303,5 +303,6 @@
 .sidebarLinkTag {
   width: 0.8rem;
   margin-right: .25rem;
+  float: right;
```

Comment:
Not other solution than using a float here ?

Reply:
Yes, there are other solutions, like flex for example, but when I apply the flex display the tags appear oddly so I don't like to touch the case of other components to just align the dot icon right

Reply:
The issue with float is not to add css, but to use a float that not works correctly on this context. It's not vertically aligned with the text. You can try to wrap the icon like in sidebarLink to align vertically and use a justify content space between on the link the keep the dot on the right.